### PR TITLE
[Testing] Update existing visual/data report helper test suite

### PR DESCRIPTION
**Pull request by zhongnansu**
*Issue #, if available:*

*Description of changes:*

Due to the previous code changes, need update existing test suites accordingly.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: kibana-reports/tsconfig.json [hunk 1]
```diff
@@ -9,12 +9,14 @@
     "module": "commonjs",
     "outDir": "./target",
     "noEmit": true,
-    "types": ["node", "jest", "react"],
```
Comment (davidcui-amzn) [inline kibana-reports/tsconfig.json:12]: Are these not needed? I thought this line had to be included

**Review by davidcui-amzn — APPROVED** [review]
LGTM - just one minor comment

File: kibana-reports/tsconfig.json (hunk 1 above)
Comment (zhongnansu) [reply kibana-reports/tsconfig.json:12]: Deleting this line will get rid of the type error on jest related types, such as `describe()`, `test`, `jest.fn()`. I am not sure about other negative affects, any idea?

Comment (davidcui-amzn) [reply kibana-reports/tsconfig.json:12]: Do tests pass when you run `yarn test --coverage`? Aside from the ones that haven't been updated.

Comment (zhongnansu) [reply kibana-reports/tsconfig.json:12]: image

Comment (zhongnansu) [reply kibana-reports/tsconfig.json:12]: I think there are existing snapshot tests that are not fixed yet

Comment (davidcui-amzn) [reply kibana-reports/tsconfig.json:12]: Ok- should be fine then. It'll be an easy one-line fix if it does end up breaking anything, anyway 👍

**Review by joshuali925 — APPROVED** [review]

**Comment by zhongnansu** [comment]
Added one more line in `visualReportHelper.ts` to pass `yarn test --coverage` , according to https://github.com/istanbuljs/nyc/issues/514

**Pull request merged**